portfolio.css

```diff
@@ -7,6 +7,61 @@
   font-family: "Poppins", sans-serif;
 }
 
+.ham-icon {
+  background: none;
+  border: none;
+}
+
+.navbar__hamb {
+  display: block;
+  font-size: 5px;
+  align-items: flex-end;
+  cursor: pointer;
+}
+
+.hidden {
+  display: none;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.display {
+  /* flex-direction: column; */
+  background: #396df2;
+
+  /* top: 0;
+  right: 0; */
+  width: 585px;
+  text-align: left;
+  height: 200px;
+
+  /* position: absolute; */
+  z-index: 2;
+  cursor: pointer;
+}
+
+.nav-items {
+  padding: 20px;
+}
```

Comment:
- [ ] I really appreciate the effort you put into designing the mobile menu but as part of the project requirements please make sure to stick to the given Figma design. This means the menu should be in its own container that spans the full-screen height with a blurred background and also make sure the color, font family & size match the Figma design as shown below 👇
- Figma design:
 ![image](https://github.com/jerymey/My-portfolio/assets/104968236/9c3dc458-0286-4873-ac48-6a797df8daae)

- Your design:
 ![image](https://github.com/jerymey/My-portfolio/assets/104968236/79a113dc-e097-4b73-b358-674f37c8b8b7)
- [ ] Also please make sure to implement the functionality of the cross icon in the menu as it should close the mobile menu.